### DISCUSSION TITLE
Typescript

### DISCUSSION THREAD

**Discussion by vagnermengali** [RCTLog/grupo7-projeto-front-end-t12#72] [Q&A] [answered]
Dei uma olhada no projeto e vê que foi toda construída em Javascript, que tal dar uma inovada e implementar Typescript, deixa tudo bem mais fácil e ainda deixa o projeto mais completo.

**Comment by gabrielfray** [comment, marked as answer]
Já implementamos no nosso projeto Vagner, mas obrigado por sua sugestão!

**Comment by alwazirconnect** [comment]
esqu'il ya un moyen de faure un chat qui utilise github en c#
